xla/backends/cpu/runtime/onednn_thunk.cc

```diff
@@ -0,0 +1,150 @@
+/* Copyright 2024 The OpenXLA Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please add a new line before and after this line.
```suggestion

    http://www.apache.org/licenses/LICENSE-2.0

```